ska_service/Dockerfile

```diff
@@ -0,0 +1,44 @@
+##############
+# Build SKA2 #
+##############
+FROM rust:1.82 as builder
```

Comment:
A reflection on the builder. Sometimes it is nice to have a less-slim, more bloated container so that there are more preinstalled tools that can be used when debugging (such as `less`, `curl`, `htop` and so on).

I realize this isn't as elegant and more space consuming.

Maybe an alternative is to identify a minimal set of frequently used debugging tools and consider installing these in the run container.

These are just loose thoughts. Something to discuss at some point.

Reply:
I generaly agree. I have considered to create a shared base image for at least all python-based bonsai related containers. A shared base could also be applied to tools like Gens